meeting-notes/2020-11-01.md

```diff
@@ -0,0 +1,65 @@
+# Adeira working group meeting 1.11.2020
+
+## Attendees
+
+- Martin
+- Trond
+- Michal
+
+## Monorepo transition to Bazel
+
+- Bazel is better with caching, knows what happened with the code, understand dependencies.
+- It will enable more languages in the monorepo.
+- No need to change Shipit or NPM publisher.
+- Currently, it runs too long in GH pipeline, that should be fine once it's done correctly.
+- Why Bazel? Most promising one, good community, features and support.
+- Once it's done right, should be a pleasure to use.
+- Trond is playing with Bazel: https://github.com/tbergq/bazel-monorepo
+
+## Vercel preview
+
+- Do we want to start using it? Yes, hopefully, it's gonna be for free.
+
+## Typescript
+
+- It's better to have them in DefinitelyTyped repo
+- Better linters and tooling there
+- Publish info about TS support in Docs or Readme
+
+## React 17
+
+- We should wait for full support in Storybook.
+- Storybook is used only in SX Tailwind, can be removed, SX Tailwind website is a better showcase
+- New JSX transform should be tried. Removes the need for `import * as React` and there is a codemod for that.
+
+## SX
+
+- It's good to have `mdn-data` as a source for generated types
+- A missing property is probably a bug with the generator
```

Comment:
I checked and it is. The missing property you were mentioning is in `mdn-data` so I have to fix the generator. Can you please open an issue?